### PR TITLE
RDOC19 Conclude Qualifiers

### PR DESCRIPTION
**Pull request by Pennek**

### CODE REVIEW

File: wiki/Tournaments/RDOC/2019/en.md [hunk 1]
```diff
@@ -59,39 +58,41 @@ The Royal Danish osu!Cup 2019 is run by various community members.
...
-| 23 | ![][flag_DK] [Decrescendo](https://osu.ppy.sh/users/12900042) |
-| 24 | ![][flag_DK] [kim1998](https://osu.ppy.sh/users/14599636) |
-| 25 | ![][flag_DK] [Cittasnaf](https://osu.ppy.sh/users/13347734) |
-| 26 | ![][flag_DK] [BPK](https://osu.ppy.sh/users/12042090) |
-| 27 | ![][flag_DK] [Tr0nixCS](https://osu.ppy.sh/users/6734261) |
-| 28 | ![][flag_DK] [Dralle](https://osu.ppy.sh/users/12380988) |
-| 29 | ![][flag_DK] [NoSync](https://osu.ppy.sh/users/8634493) |
-| 30 | ![][flag_DK] [Slayco](https://osu.ppy.sh/users/9449330) |
-| 31 | ![][flag_DK] [Liquidminds](https://osu.ppy.sh/users/13438435)
+| Seed | Members | Qualifier Score |
```
Comment (cl8n) [inline wiki/Tournaments/RDOC/2019/en.md]:
```suggestion
| Seed | Player | Qualifier score |
```

File: wiki/Tournaments/RDOC/2019/en.md [hunk 2]
```diff
@@ -59,39 +58,41 @@ The Royal Danish osu!Cup 2019 is run by various community members.
...
+| :-- | :-- | :-: |
```
Comment (cl8n) [inline wiki/Tournaments/RDOC/2019/en.md]:
```suggestion
| --: | :-- | --: |
```
I think right-aligned is nice for numbers meant to be compared against each other, that way you can easily see when there are more significant digits

File: wiki/Tournaments/RDOC/2019/en.md [hunk 3]
```diff
@@ -16,7 +16,6 @@ The **Royal Danish osu!Cup 2019** (***RDOC 2019***) is an ongoing Danish 1v1 dou
 | --: | :-- |
 | Registration phase | 2020-01-10/2020-01-24 |
 | Qualifiers | 2020-02-07/2020-02-16 |
-| Live drawings | *to be determined* |
```
Comment (cl8n) [inline wiki/Tournaments/RDOC/2019/en.md]: didn't happen?

Comment (Pennek) [reply wiki/Tournaments/RDOC/2019/en.md]: nope, since the format was changed from groupstage to ro32 there needs to be no live drawings

**Pull request merged**